tests/test_networks.py

```diff
@@ -315,6 +316,19 @@ class Model(BaseModel):
     assert Model(v=input).v.tld == output
 
 
+@pytest.mark.parametrize(
+    'value',
+    [
+        'file:///foo/bar',
```

Comment:
Shouldn't we also check `file://localhost/foo/bar`? And possibly `file:////host/foo/bar`, see [File_URI_scheme#How_many_slashes?](https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes?)

Reply:
added the test (with both 2 and 4 slashes. Also added a `file://localhost/foo/bar` test for `test_any_url_success`